core/includes/menu.inc

```diff
@@ -636,7 +636,7 @@ function _menu_load_objects(&$item, &$map) {
           $return = call_user_func_array($function, $args);
         }
         else {
-          $return = $function($value);
+          $return = ($value[0] === '%') ? NULL : $function($value);
```

Comment:
Isn't `$value` a string here? If so, let's avoid using the array-syntax to get the first character, that type of string checking is error-prone when unexpected values are passed (e.g. it operates on an array unexpectedly). I'd prefer `substr()` or even `backdrop_substr()` here, even though it's more verbose.

Reply:
I'll change it to use`backdrop_substr()`.